2-ui/4-forms-controls/1-form-elements/article.md

```diff
@@ -246,7 +246,7 @@ This syntax is optional. We can use `document.createElement('option')` and set a
 
 The difference between `defaultSelected` and `selected` is that `defaultSelected` sets the HTML-attribute (that we can get using `option.getAttribute('selected')`, while `selected` sets whether the option is selected or not.
 
-In practice, we usually should set both values to `true` or `false` (or omit, that's the same as `false`).
+In practice, you should usually set _both_ values to `true` or `false` (or omit; that's the same as `false`).
```

Comment:
you → one.
And please no `_`. This formatting is never used in the tutorial.

Also are sure about ; ?

Reply:
I am not sure about the semi-colon.  But I *am* sure that it should definitely not be a comma.  That is a comma-splice. :)

Reply:
Changed “you” to “one”.  Rewrote the statement about omission.